NitroxClient/GameLogic/InitialSync/GlobalRootInitialSyncProcessor.cs

```diff
@@ -16,7 +18,7 @@ public GlobalRootInitialSyncProcessor(Entities entities)
         public override IEnumerator Process(InitialPlayerSync packet, WaitScreen.ManualWaitItem waitScreenItem)
         {
             Log.Info($"Received initial sync packet with {packet.GlobalRootEntities.Count} global root entities");
-            yield return entities.SpawnAsync(packet.GlobalRootEntities);
+            yield return entities.SpawnAsync(packet.GlobalRootEntities.Cast<Entity>().ToList());
```

Comment:
Is this cast necessary?

Reply:
Yes, entities in global root are `WorldEntity`.  C# won't autocast the collection to the base collection.

Reply:
I take it back.  After the merge, this is now an `Entity` (due to the serialization issues with `List`).  Updated.